skimage/measure/_regionprops.py

```diff
@@ -941,19 +941,28 @@ def regionprops_table(
     extra_properties=None,
     spacing=None,
 ):
-    """Compute image properties and return them as a pandas-compatible table.
+    """Compute region properties and return them as a pandas-compatible table.
 
-    The table is a dictionary mapping column names to value arrays. See Notes
-    section below for details.
+    The return value is a dictionary mapping property names to value arrays.
+    This dictionary can be used as input to ``pandas.DataFrame`` to result in
+    a "tidy" [1]_ table with one region per row and one property per column.
```

Comment:
This seems a bit redundant with the description in "Returns", no? Since the first line already implies pandas I'd remove this section in favor of highlighting the rest of the docstring.

```suggestion
```

Reply:
(Not a blocker though!)

Reply:
I agree it's slightly redundant... Would you mind if I replaced "a pandas-compatible table" by "tabular data" in the first line?

Reply:
I'd like the tidy reference to remain somewhere. It's not present in the Return section currently.

Reply:
I didn't mean that this is verbatim included in returns. But I thought addressing this aspect in detail would make more sense in the return section where Pandas' DataFrame is already referenced.